hw/ip/spi_host/data/spi_host.hjson

```diff
@@ -0,0 +1,590 @@
+                   CPHA = 0 data is both shifted the trailing edge, and sampled
+                   a full cycle later.  If CPHA = 1 data is shifted and
+                   sampled with the trailing edge, also separated by a
+                   full cycle.'''
+          resval: 0
+        },
+        { bits: "27",
+          name: "CPHA",
+          desc: '''The phase of the sck clock signal relative to the data. When
+                   CPHA = 0, the data changes on the trailing edge of sck
+                   and is typically sampled on the leading edge.  Conversely
+                   if CPHA = 1 high, data lines change on the leading edge of
+                   sck and are typically sampled on the trailing edge.
+                   CPHA should be chosen to match the phase of the selected
+                   device.  The sampling behavior is modified by the
+                   FLCYC bit.
+                   ''',
+          resval: "0x0"
+        },
+        { bits: "26",
+          name: "CSAAT",
+          desc: '''Chip select active after transaction.  If CSAAT = 0, the
+                   chip select line is raised immediately at the end of the
+                   transaction.   If CSAAT = 1, the chip select line is left
+                   low at the end of the current sequence.  This potentially
+                   allows for the creation of longer or otherwise more
+                   complex sequences.'''
+          resval: "0x0"
+        },
+        { bits: "25:23",
+          name: "CSNLEAD",
+          desc: '''CS_N Leading Time.  Indicates the number of half sck cycles,
+                   CSNLEAD+1, to leave between the falling edge of cs_n and
+                   the first edge of sck.  Setting this register to zero
+                   corresponds to the minimum delay of one-half sck cycle'''
+          resval: 0
+        },
+        { bits: "22:20",
+          name: "CSNTRAIL"
+          desc: '''CS_N Trailing Time.  Indicates the number of half sck cycles,
+                   CSNTRAIL+1, to leave between last edge of sck and the rising
+                   edge of cs_n. Setting this register to zero corresponds
+                   to the minimum delay of one-half sck cycle.'''
+          resval: 0
+        },
+        { bits: "19:17",
+          name: "CSNIDLE"
+          desc: '''Minimum idle time between commands. Indicates the minimum
+                   number of sck half-cycles to hold cs_n high between commands.
+                   Setting this register to zero creates a minimally-wide CS_N-high
+                   pulse of one-half sck cycle.'''
+          resval: 0
+        },
+        { bits: "15:0",
+          name: "CLKDIV"
+          desc: '''Core clock divider.  Slows down subsequent SPI transactions by a
+                   factor of (CLKDIV+1) relative to the core clock frequency.  The
+                   period of sck, T(sck), then becomes 2*(CLK_DIV+1)*T(core)'''
```

Comment:
Since this text is interpreted as markdown, I think it needs some back-ticks (at the moment, the multiplication symbols disappear and the middle term gets italicized!)

Reply:
I spoke too soon: this isn't quite interpreted as proper markdown; we've got a sort of re-implementation of some of markdown plus the !! syntax, and it didn't include back-tick support. I've just added that in PR #5116, so it should hopefully be ready soon!

Reply:
Hmm.   I'll have to come back to this comment after tackling the rest.

Reply:
Thanks Rupert, I rebased your doc changes on to this PR and the results look great!  Backticks added.